CONTRIBUTING.md

```diff
@@ -70,10 +70,14 @@ guidelines:
 
 ### Commits in your pull-requests should
 
-- Have a useful description prefixed with the package name (E.g.: "foopkg: Add
-  libzot dependency")
+- Have a useful commit subject prefixed with the package name (E.g.: "foopkg:
+  Add libzot dependency").
 - Include Signed-off-by tag in the commit comments.  See: [Sign your
   work](https://openwrt.org/submitting-patches#sign_your_work)
+- Author and sign-off must match and be a real name or known identity and
+  a real email address. Github private email addresses will not be accepted.
```

Comment:
```suggestion
  a real email address. GitHub private email addresses will not be accepted.
```

Reply:
Resolved Github -> GitHub in my new work as well as 6 other references in the file. Also updated Sourceforge -> SourceForge, because I happened to see it when I was resolving one of the GitHub instances.